app/src/pages/PageJoinNetwork.re

```diff
@@ -1,34 +1,38 @@
 open Atom;
-open Layout;
 open DesignSystem;
 
 module Styles = {
   open Css;
 
+  let content =
```

Comment:
Could this be moved out to the Layout in the DesignSystem?

Reply:
For sure, I forgot to do it in the rest of the codebase.